server/indexer_service.go

```diff
@@ -0,0 +1,111 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package server
+
+import (
+	"context"
+	"time"
+
+	"github.com/cometbft/cometbft/libs/service"
+	rpcclient "github.com/cometbft/cometbft/rpc/client"
+	"github.com/cometbft/cometbft/types"
+	evmostypes "github.com/evmos/os/types"
```

Comment:
why not
```suggestion
	ostypes "github.com/evmos/os/types"
```
?

Reply:
technically it's still "evmOStypes" but all in small letters 😅